cfme/tests/pod/test_appliance_crud.py

```diff
@@ -572,7 +573,7 @@ def test_pod_appliance_db_backup_restore(temp_pod_appliance, provider, setup_pro
     # check that appliance is running and provider is available again
     collection = appliance.collections.container_projects
     proj = collection.instantiate(name=appliance.project, provider=provider)
-    assert navigate_to(proj, 'Dashboard')
+    assert navigate_to(proj, 'Dashboard', wait_for_view=120)
```

Comment:
120 seconds seems like a long time to navigate to the project dashboard page. Why so long?